instrumentation/runtime-telemetry-jfr/library/build.gradle.kts

```diff
@@ -0,0 +1,79 @@
+plugins {
+  id("otel.library-instrumentation")
+  id("java-test-fixtures")
+}
+
+otelJava {
+  minJavaVersionSupported.set(JavaVersion.VERSION_17)
```

Comment:
Curious how this works with the agent. Do we publish different versions of the agent, or do we package the jfr stuff up in the java 8 agent and only allow it to be activated if we detect we're running on java 17+?

Reply:
Discussed in the [March 2 2023 SIG ](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/edit#heading=h.7khhmsx7w18m). Confirmed nothing must be done in order to prevent JDK17 features from breaking things.